js/index.js

```diff
@@ -468,9 +508,22 @@ webix.ready(function(){
     }
 	}
 
-  //sync from list to chart
+//Sync operations
+  users_list.sync(
+    users_data_collection,
+    function(){
+      users_list.filter(function(obj){
```

Comment:
looks strange. .filter() is used not for filtering, but for injecting obj.$css.
you can freely add it to the collection scheme and only list will use this rule

Reply:
It was changed

Reply:
as far as I can see, `scheme` is also defined for a user list synced with this collection. but it is not working there, because there can be only one scheme per datastore. so it can be removed from the list

Reply:
I know, it doesn't work. Just forgot to delete. It was changed